requirements.txt

```diff
@@ -24,6 +25,7 @@ nose==1.3.7
 numpy==1.16.4
 packaging==19.0
 pandas==0.25.0
+pkg-resources==0.0.0
```

Comment:
This dependency is an artifact of a bug in Ubuntu, as @orionw discovered. Make sure you do not include it in the `requirements.txt`. See https://stackoverflow.com/questions/39577984/what-is-pkg-resources-0-0-0-in-output-of-pip-freeze-command.

Reply:
For some reason, pkg-resources automatically installs itself when I create a new virtual environment. I'll make sure to uninstall it from any environments that I use.

Reply:
Yes, the reason is the bug I linked. When you run `pip3 freeze > requirements.txt`, just manually remove that line before you git add/commit. It won't hurt your environment.